src/mame/drivers/ksys573.cpp

```diff
@@ -5951,9 +6109,6 @@ ROM_START( kicknkick )
 	ROM_LOAD( "a36eaa.31h",   0x000000, 0x200000, CRC(6475da5f) SHA1(6b1de2ed06504583bfa3e5b11851c459d227e6d8) )
 	ROM_REGION( 0x200000, "29f016a.27h", 0 ) /* onboard flash */
 	ROM_LOAD( "a36eaa.27h",   0x000000, 0x200000, CRC(1179ab7b) SHA1(19a316cacb6eb87b905884091820e6b53aef64b7) )
-
-	ROM_REGION( 0x002000, "m48t58", 0 )
-	ROM_LOAD( "a36eaa.22h",   0x000000, 0x002000, CRC(d193e1b6) SHA1(e201c7fdce08d512a9a5dd429b35db907f557c03) )
```

Comment:
The Kick & Kick machine had a dead RTC so I sent over a clean RTC which was generated with MAME and it got included in the ROM data here. Kick & Kick will initialize the RTC on its own when you start the game so I don't think it's necessary to include a MAME initialized RTC in the ROM set.

![0011](https://user-images.githubusercontent.com/63495610/158026445-b8dfc351-5650-4a6e-8d91-257c996bd836.png)